plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -1239,14 +1239,18 @@ $.extend(DataTable.prototype, UIControl.prototype, {
             };
         };
 
-        var getText = function (text, addDefault) {
-            text = _pk_translate(text);
-            if (text.indexOf('%s') > 0) {
-                text = text.replace('%s', '<br /><span class="action">&raquo; ');
+        var getText = function (text, addDefault, replacement) {
+            if (/(%(.\$)?s+)/g.test(_pk_translate(text))) {
+                var values = ['<br /><span class="action">&raquo; '];
+                if(replacement) {
+                    values.push(replacement);
```

Comment:
I haven't tested it yet but does it actually do the same? I think before it was maybe replacing the first `%s` with `self.props.pivot_dimension_name` and now it's maybe the second `%s` (`replacement`)